gbasis/evals/eval_deriv.py

```diff
@@ -147,6 +147,17 @@ def evaluate_deriv_basis(
 ):
     r"""Evaluate the derivative of the basis set in the given coordinate system at the given points.
 
+    The derivative (to arbitrary orders) of a basis function is given by:
+
+    .. math::
+    
+        \frac{\partial^{m_x + m_y + m_z}}{\partial x^{m_x} \partial y^{m_y} \partial z^{m_z}}
+        \phi(\mathbf{r}_n | \mathbf{R}_{A}, \mathbf{a}_j, \mathbf{d}_k, \boldsymbol{\alpha})
```

Comment:
We can remove all other arguments of $\phi$ except $\mathbf{r}$ like when the basis functions appear in integrals and evaluations. It makes things simpler. What do you think?

Reply:
I made the change here 3fdabbcd1e70c149f8f4a40b65a07ddf61d0c1ae, together with the missing sum for the point charge.